libr/asm/arch/8051/8051_ops.h

```diff
@@ -213,5 +213,40 @@ static _8051_op_t _8051_ops[] = {
 	{0xff, OP_INVALID, NULL, 0, 0, 0}
 };
 
+static const char *_8051_regs[] = {
+	"R0", "R1", "R2", "R3", "R4", "R5", "R6", "R7", // 0x00
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x08
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x10
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x18
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x20
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x28
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x30
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x38
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x40
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x48
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x50
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x58
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x60
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x68
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x70
+	0, 0, 0, 0, 0, 0, 0, 0, // 0x78
+	"P0", "SP", "DPL", "DPH", 0, 0, 0, "PCON", // 0x80
```

Comment:
we should put all those register names in lowercase. otherweise the asm.ucase filter will not be honored. this happens in all archs in r2land.